conf/iridanext.config

```diff
@@ -5,11 +5,7 @@ iridanext {
         overwrite = true
         files {
             idkey = "irida_id"
-            global = [
-                "**/pipeline_info/software_versions.yml",
-                "**/locidex/concat/query/MLST_error_report_concat_query.csv",
-                "**/locidex/concat/reference/MLST_error_report_concat_ref.csv"
```

Comment:
Maybe I missed the discussion on this, but does this mean that users who run the pipeline manually on Irida Next won't have these files anymore? Is gasnomenclature run manually?